pkg/neorpc/result/version_test.go

```diff
@@ -42,6 +42,10 @@ func TestVersion_MarshalUnmarshalJSON(t *testing.T) {
             "validatorscount": 7,
             "hardforks": [{"name": "Aspidochelone", "blockheight": 123}, {"name": "Basilisk", "blockheight": 1234}]
         },
+        "rpc": {
+            "maxiteratorresultitems": 0,
+            "sessionenabled": false
+        },
```

Comment:
Can't be 0, add some meaningful values.

Reply:
Also need to adjust RPC client test (`pkg/rpcclient/rpc_test.go`). Better with some real response from C# if you're able to build C# node from source.